Core/BinaryViewFile.cpp

```diff
@@ -15,6 +15,16 @@ BinaryViewFile::BinaryViewFile(BinaryViewRef bv)
     : m_bv(bv)
     , m_reader(BinaryNinja::BinaryReader(bv))
 {
+	m_ptrSize = bv->GetAddressSize();
+}
+
+uint64_t BinaryViewFile::readPointer()
+{
+	if (m_ptrSize == 8)
+		return readLong();
+	else if (m_ptrSize == 4)
+		return readInt();
+	return readLong();
```

Comment:
This can be consolidated into `if (m_ptrSize == 4) readInt()` otherwise, `readLong()`